kvm.yaml

```diff
@@ -8,7 +30,7 @@
       version: 0.7.0
     - name: cluster-operator
       provider: kvm
-      version: 0.22.1-dev
+      version: 0.21.4-dev
```

Comment:
Downgrade?

Reply:
Probably I was about to make it the same as a legacy version in https://github.com/giantswarm/cluster-operator/blob/397bbf0076dd67b80aaad8e734a7d1bafd307a8e/pkg/project/project.go#L4
during my last manual tests. 

But it's already up to 0.21.5-dev, I will update it when this PR is out of the draft.